app.js

```diff
@@ -1,13 +1,24 @@
 const express = require('express');
 const app = express();
-const { getTopics } = require('./controller')
+const { getTopics, getArticles } = require('./controller')
 
 app.use(express.json());
 
```

Comment:
Why would you have this before a post or patch I wonder???? 🤔 📸

Reply:
I just put it in at the beginning because I know I would forget to put it in later down the line